lib/fetch.js

```diff
@@ -44,6 +44,10 @@ export function mapAndSanitate(item, type) {
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
       mapDate(item.data[`${type}.timestamp`].value) : null,
+    startDateTime: item.data[`${type}.timestamp`] ?
```

Comment:
Whilst you're at it, could you maybe put [date formatting logic](https://github.com/redbadger/badger-brain/blob/master/lib/fetch.js#L5) into a [resolver for 'datetime'](https://github.com/redbadger/badger-brain/blob/master/lib/eventFields.js#L92)? This logic doesn't belong here, IMO.